src/client/js/Layouts/DefaultLayout/DefaultLayoutConfig.json

```diff
@@ -13,6 +13,12 @@
       "DEBUG_ONLY": false
     },
     {
+      "id": "ObjectBrowserPanel",
```

Comment:
Yes, if it's not before the Visualizers the locate button will be to the right of the visualizer's button at initial load and when the visualizer is switched - the locate button will be to the left.
This way it's always to the left.